packages/microsoft_exchange_online_message_trace/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -14,6 +14,9 @@ processors:
   - json:
       field: event.original
       target_field: microsoft.online_message_trace
+  # Drop "null" event
+  - drop:
+      if: ctx.microsoft?.online_message_trace?.value != null
```

Let's make the test more specific.
```suggestion
      if: ctx.microsoft?.online_message_trace?.value instanceof List && ctx.microsoft.online_message_trace.value.length == 0 
```